advanced/enable_extensions.md

```diff
@@ -28,3 +28,17 @@ activation commands in your user enviroment script.
 [nbextensions]: ../images/nbextensions.png "Nbextensions configuration panel"
 [terminal_ext]: ../images/terminal_ext.png "Extension activation on terminal"
 [spellchecker_nb]: ../images/spellchecker_nb.png "Extension activation inside a notebook"
+
+
+# Enable Jupyter extension not provided by SWAN.
+
+Install Jupyter extension requires to run the next commands from a SWAN terminal:
```

Comment:
Proposal: `It is also possible to install and enable a Jupyter extension which is not provided by default by SWAN. The following commands show how that can be done, from the SWAN terminal:`